src/routes/adm/router.js

```diff
@@ -1,7 +1,9 @@
 const express = require('express')
 const router = express.Router()
-
 const register = require('./register')
+const csvRegisterUser = require('./csvRegisterUser')
+const multer = require('multer')
+const upload = multer({ dest: '/tmp' })
```

Comment:
multer.memoryStorage is not working ?

Reply:
I figured out i need to store the file to import it, so this way is the best i found

Reply:
Maybe try to check if it's possible to add a file limit.

We don't want people sending 100Go files to the server and saturating it